### DISCUSSION TITLE
Email notification for login in another system

### DISCUSSION THREAD

**Discussion by keshva23** [cvat-ai/cvat#4055] [Q&A]
If the user tries to login with same url in another system, it should send an email notification to the user.

how can we implement this feature?

**Comment by bsekachev** [comment]
Hi, 
Unfortunately I don't understand your idea. Could you clarify?

**Reply by keshva23** [reply]
@bsekachev  Let me correct the query.    How we can add the session in CVAT application, once user logs in?

**Reply by bsekachev** [reply]
What is ``session`` in your context?

**Reply by keshva23** [reply]
@bsekachev Means , when a user logs in, session has to be created .
